src/common/popup/components/about.jsx

```diff
@@ -18,7 +18,7 @@ function About() {
               <img
                 src="../icons/icon-32.png"
                 className="logo-sm"
-                role="presentation"
+                alt=""
```

Comment:
what was the purpose of `role` before? why do we need an empty `alt` attribute?
Shouldn't `alt`'s always have content describing the image? (I'm not sure on all these questions, its not meant as passive aggressive questions :) )

Reply:
https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/alt-text.md#how-to-resolve

Reply:
Since it's an icon, supposedly it's okay to set an empty `alt` attribute according to `eslint-plugin-jsx-a11y`.

Reply:
You are all over the place recently @tessi 😁

Reply:
:D 
if you notice that, you must be all over the place too ;)

Reply:
Nah, I just get all your GitHub notifications 😛